src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="imac with colorful background"
+      >
+
+      <div class="card__body">
+        <h1 class="card__body-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <p class="card__body-description">
+          Product code: 195434
+        </p>
+
+        <div class="card__body-rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__body-rating-reviews">
+            Reviews: 5
+          </p>
+        </div>
```

Comment:
I'd recommend to add an empty line either between some logic blocks (when you use JS) or _multiline_ siblings (not _oneline_ ones) to increase readability
```suggestion
        <div class="card__body-rating">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
          
          <p class="card__body-rating-reviews">
            Reviews: 5
          </p>
        </div>
```